protoc-gen-swagger/genswagger/types.go

```diff
@@ -57,6 +57,7 @@ type swaggerObject struct {
 	Produces            []string                            `json:"produces"`
 	Paths               swaggerPathsObject                  `json:"paths"`
 	Definitions         swaggerDefinitionsObject            `json:"definitions"`
+	StreamDefinitions   swaggerDefinitionsObject            `json:"streamdefinitions"`
```

Comment:
I can't find a reference for this key in the [OpenAPI v2 spec](https://swagger.io/specification/v2/); is this supported by the spec or an extensions?

Reply:
oh shoot you're right from looking it over a little closer I think I have a few options:
1. change `streamdefinitions` to `x-stream-definitions` which is the allowed way to extend the Swagger Schema.
2. add the stream result wrapped messages under definitions with a special name that cannot be generated from a proto field like `stream*TestStreamMessage` and referenced as `#/definitions/stream*TestStreamMessage` ie
```json
  "definitions": {
    "stream*TestStreamMessage": {
      "type": "object",
      "properties": {
        "result": {
          "$ref": "#/definitions/TestStreamMessage"
        }
      },
      "title": "Stream result of TestStreamMessage"
    },
    "TestStreamMessage": {
      "type": "object",
      "properties": {
        "test": {
          "type": "string"
        }
      }
    }
  }
```
Both options work and follow the OpenAPI v2 spec from my understanding, any opinions of which we should go with?

Reply:
I like the former better :).

Reply:
ok ill make the change to `x-stream-definitions` and figure out what to do about a test